mvt/ios/modules/mixed/viber.py

```diff
@@ -0,0 +1,154 @@
+# Mobile Verification Toolkit (MVT)
+# Copyright (c) 2021-2023 Claudio Guarnieri.
+# Use of this software is governed by the MVT License 1.1 that can be found at
+#   https://license.mvt.re/1.1/
+
+import logging
+import sqlite3
+import fnmatch
+from typing import Optional, Union
+
+from mvt.common.utils import check_for_links, convert_mactime_to_iso
+
+from ..base import IOSExtraction
+
+VIBER_BACKUP_IDS = [
+    "83b9310399a905c7781f95580174f321cd18fd97",
+]
+VIBER_ROOT_PATHS = [
+    "private/var/mobile/Containers/Shared/AppGroup/*/viber-database.sqlite",
```

Comment:
Not 100% sure on this, would need confirmation with an IOS backup containing Viber data